js/src/popover.js

```diff
@@ -121,6 +127,60 @@ class Popover extends Tooltip {
     tip.classList.remove(ClassName.SHOW)
   }
 
+  _getConfig(config) {
+    const dataAttributes = Manipulator.getDataAttributes(this.element)
+
+    Object.keys(dataAttributes)
+      .forEach(dataAttr => {
+        if (DISALLOWED_ATTRIBUTES.indexOf(dataAttr) !== -1) {
+          delete dataAttributes[dataAttr]
+        }
+      })
+
+    if (config && typeof config.container === 'object' && config.container.jquery) {
+      config.container = config.container[0]
+    }
+
+    config = {
+      ...this.constructor.Default,
+      ...dataAttributes,
+      ...typeof config === 'object' && config ? config : {}
+    }
+
+    if (typeof config.delay === 'number') {
+      config.delay = {
+        show: config.delay,
+        hide: config.delay
+      }
+    }
+
+    if (typeof config.title === 'number') {
+      config.title = config.title.toString()
+    }
+
+    if (typeof config.content === 'number') {
+      config.content = config.content.toString()
+    }
+
+    typeCheckConfig(
+      NAME,
+      config,
+      this.constructor.DefaultType
+    )
+
+    if (config.sanitize) {
+      config.template = sanitizeHtml(config.templateWithHeader, config.whiteList, config.sanitizeFn)
+      console.log(this.element.getAttribute('title') === null)
+      console.log(config.title === null)
+      console.log('--------------------------')
```

Comment:
Weird that this passes linting. I'll make an issue but until then drop this

Reply:
Sorry for this oversight! I deleted these dump lines.